.github/workflows/testacc.yml

```diff
@@ -0,0 +1,63 @@
+name: Acceptance Test
```

Comment:
Could you add license header (see other files)

Reply:
done